src/main/java/com/skeleton/feed/service/PostService.java

```diff
@@ -72,11 +77,30 @@ public AddLikeResponse addLike(Long id) {
         return new AddLikeResponse(post);
     }
 
+
+    public PostDetailResponse getPostDetail(Long id) {
+        Post post = getPost(id);
+        incrementViewCount(id);
+        return new PostDetailResponse(post);
+    }
+
+    public Post getPost(Long id) {
```

Comment:
```suggestion
    private Post getPost(Long id) {
```
`외부에서 직접 호출하지 않고`, `Post 엔티티를 리턴`하는 이런 메서드는 private가 좋을 것 같습니다.  마찬가지로 `incrementViewCount` 도 private여야합니다.

Reply:
넵 수정하겠습니다!